model/schema/core.yaml

```diff
@@ -893,6 +921,21 @@ slots:
     range: string
     required: true
 
+  source_organization:
+    domain: DataProvider
+    range: Organization
+    required: true
+
+  source_organization_abbreviation:
+    domain: DataProviderDTO
+    range: string
+    required: true
+
+  resource_descriptor_page:
+    domain: CrossReference
+    range: ResourceDescriptorPage
+    required: true
```

Comment:
@oblodgett - I think we need to make this non-required otherwise we would have to create ResourceDescriptor and ResourceDescriptorPage entries for any organisation that the ontologies have xrefs to

Reply:
Yes I agree.